File: frontend/server/models/Therapist.js
const Sequelize = require('sequelize');
const sequelize = require('../config/sequelize');

const Therapist = sequelize.define('Therapist', {
    name: { type: Sequelize.STRING, allowNull: false },
    number: { type: Sequelize.INTEGER, allowNull: false },
    email: { type: Sequelize.STRING, allowNull: false },
    location: { type: Sequelize.STRING, allowNull: true },
    description: { type: Sequelize.STRING, allowNull: false }
});


module.exports = Therapist;

File: frontend/lib/models/therapist.dart
class Therapist {
  int id;
  String name;
  int number;
  String email;
  String location;
  String description;

  Therapist(this.id, this.name, this.number, this.email, this.location,
      this.description);

  Therapist.fromJson(Map<String, dynamic> parsedJson) {
    this.id = parsedJson['id'];
    this.name = parsedJson['name'];
    this.number = parsedJson['number'];
    this.email = parsedJson['email'];
    this.location = parsedJson['location'];
    this.description = parsedJson['description'];
  }
}
